src/utah-design-system/Select.jsx

```diff
@@ -0,0 +1,78 @@
+import * as RadixSelect from '@radix-ui/react-select';
+import PropTypes from 'prop-types';
+import { twMerge } from 'tailwind-merge';
+import Icon from './Icon';
+
+export default function Select({
+  className,
+  disabled,
+  items,
+  onValueChange,
+  open,
+  placeholder,
+  value,
+}) {
+  return (
+    <RadixSelect.Root
+      open={open}
+      value={value}
+      onValueChange={onValueChange}
+      disabled={disabled}
+    >
+      <RadixSelect.Trigger
+        className={twMerge(
+          'group flex h-9 w-full items-center justify-between rounded-md border border-slate-400 px-2 py-1',
+          disabled && 'cursor-not-allowed text-slate-300',
+          'data-[placeholder]:text-slate-500',
```

Comment:
**thought**: placeholder text doesn't need to be a data attribute. https://tailwindcss.com/docs/hover-focus-and-other-states#placeholder-text

Reply:
That doesn't work because the placeholder is actually in the value component below. This is just a [help for styling](https://www.radix-ui.com/docs/primitives/components/select#with-a-placeholder:~:text=There%27s%20also%20a%20data%2Dplaceholder%20attribute%20on%20Trigger%20to%20help%20with%20styling.).